test-alerts.cpp

```diff
@@ -5,4 +5,17 @@
 
 TEST_CASE("infers the breach according to limits") {
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
+  REQUIRE(inferBreach(35, 20, 30) == TOO_HIGH);
+  REQUIRE(inferBreach(25, 20, 30) == NORMAL);
+  
+  //REQUIRE(TypesOfLimitCooling(PASSIVE_COOLING) == (struct coolingLimitType Limit={0,35}));
+  //REQUIRE(TypesOfLimitCooling(HI_ACTIVE_COOLING) == (coolingLimitType Limit= {0,45}));
+  //REQUIRE(TypesOfLimitCooling(MED_ACTIVE_COOLING) == (coolingLimitType Limit= {0,40}));
+  
+  REQUIRE(classifyTemperatureBreach(PASSIVE_COOLING, 12) == 0);
```

Comment:
Test cases needs to be extended to cover all cases including email and controller target. Please add tests.